src/theme.tsx

```diff
@@ -210,38 +210,38 @@ export type EventFormat = 'universal-analytics' | 'google-analytics-4';
 
 // This is the type expected by ThemeContextProvider theme prop.
 // This config is provided by the user of the lib
-export type ThemeConfig = {
+export type ThemeConfig = Readonly<{
     skin: Readonly<Skin>;
     colorScheme?: ColorScheme; // light by default. TODO: Change to auto by default in next major version
-    i18n: {
+    i18n: Readonly<{
         locale: Locale;
         phoneNumberFormattingRegionCode: RegionCode;
-    };
-    platformOverrides?: {
+    }>;
+    platformOverrides?: Readonly<{
         platform?: 'ios' | 'android' | 'desktop';
         insideNovumNativeApp?: boolean;
         userAgent?: string;
-    };
-    texts?: Partial<ThemeTexts>;
-    analytics?: {
+    }>;
+    texts?: Partial<Readonly<ThemeTexts>>;
+    analytics?: Readonly<{
         logEvent: (trackingEvent: TrackingEvent) => Promise<void>;
         eventFormat?: EventFormat;
-    };
-    dimensions?: {headerMobileHeight: number | 'mistica'};
+    }>;
+    dimensions?: Readonly<{headerMobileHeight: number | 'mistica'}>;
```

Comment:
This will require changes in webapp:
![image](https://user-images.githubusercontent.com/579417/201065891-651ea9e1-1fdb-44e3-9680-582aa36c0d41.png)

Reply:
That is strange, because it tells the caller that Mística requires your object to be mutable (that means that you should expect Mística manipulating your objects).
Is there any other way we can pass the initial theme config to _pages_ without changing it afterwards that way?

Reply:
Let me change the question. Do we *really* need a readonly here?
To me the important type that mistica users shouldn't be allowed to change is `Theme` (line 247).

Reply:
That is what I did here: https://github.com/Telefonica/webapp/pull/2303

No, it is not really needed until it is (the same reasoning we already had and still have with components like the `Stepper.steps`).